composables/useScaledDataByGrainType.ts

```diff
@@ -0,0 +1,16 @@
+import { group } from 'd3'
+
+export function useScaledDataByGrainType() {
+  const scaledData = useScaledGrainData();
+
+  return computed(() => {
+    // remove groupings where the key includes the word "dynamics"
```

Comment:
Not to be nitpicky, but this isn't really the correct documentation for what is happening here- you are grouping the data by grain type, and then below deleting keys that include dynamics, this makes it seem like the entire purpose of doing this is to delete, when in fact the entire purpose is to group, and deleting is an additional thing that happens

Reply:
right, i just left it in there so it was backwards compat. we could remove the dynamics ones at an earlier step, perhaps in the `useGrainData` composable

Reply:
It's not about being backwards compatible or not, it's about having documentation that describes the code that is around it, this could be really confusing to someone because the comment does not really describe what is happening- I would rather have no documentation than old documentation that is incorrect